### ISSUE TITLE
Reasons behind permissions

### ISSUE THREAD

**Issue by deecewan** [toolmantim/release-drafter#52]
Hey,

Just went to add this to an org-level repo. Noticed that it required the 'write code' permission. That's a little daunting.

Is there a list of reasons behind the permissions required?

**Comment by toolmantim** [comment]
Yep, you can find a big warning notice on the bottom of https://github.com/apps/release-drafter

GitHub doesn’t have a “Write Releases” permission, just a full code write one 😞

I consider this is a GitHub bug, and we should keep this issue open until it’s fixed (if they add a new finer grained scope).

**Comment by toolmantim** [comment]
Do you have any suggestions for where that notice could live so you would have seen it?

**Comment by deecewan** [comment]
I mean, I looked in the README.md to see if it was in there. I guess others will do the same if they're not comfortable with the permission.

Thanks for the response, tho!

**Comment by toolmantim** [comment]
Thanks @deecewan. I've added a new [GitHub Installation Permissions](https://github.com/toolmantim/release-drafter#github-installation-permissions) section to the readme and added a little more explanation as to why it's not a smaller scope.
